modules/querying/pages/accumulators.adoc

```diff
@@ -1503,6 +1503,11 @@ GroupByAccum<type [, type]* , accumType [, accumType]* >
 In the EBNF above, the *type* terms form the key set, and the *accumType* terms form the map's value. Since they are accumulators, they perform a grouping. Like a MapAccum, if we try to store a (key->value) whose key has already been used, then the new value will accumulate to the data which is already stored.
 In this case, each field of the multiple-field value has its own accumulation function. One way to think about GroupByAccum is that each unique key is a group ID.
 
+[WARNING]
+====
+Currently, operations between different GroupByAccums with the same data types but different field names are not supported. These queries may pass semantic checks but will fail during query compilation with a Type Check Error.
```

Comment:
In 4.2.0, it should be blocked by a type check error. This should be better to be added in the version < 4.2.0.